### ISSUE TITLE
System Admin Office View

### ISSUE THREAD

**Issue by jpye-finch** [opencrvs/opencrvs-core#1662]
### Problem statement
(Describe the problem you're trying to solve by doing this work)

### Proposed work
- Each office will have its own page
- See a list of all users in the office
- I click on the submenu and edit the user details

### Success criteria
(The criteria that must be met in order to consider this project a success)

### User stories
As a user, I want to see a dashboard of operational indicators, so that I can review our performance

### Scope
- Office view for System Admin
- Edit user details in one go

Future work
- Have specific journey for different changes required
- Add the user profiles images
- Click on a user name will navigate to the User Profile page
- Add status of users in chosen administrative area
- Sorting

**Comment by euanmillar** [comment]
Blocked by: https://github.com/opencrvs/opencrvs-core/issues/2256
